ui/StatusQ/src/stringutilsinternal.cpp

```diff
@@ -49,3 +50,19 @@ QString StringUtilsInternal::plainText(const QString& htmlFragment) const
 {
     return QTextDocumentFragment::fromHtml(htmlFragment).toPlainText();
 }
+
+static QKeySequence variantToKeySequence(const QVariant &var)
+{
+    if (var.metaType().id() == QMetaType::Int)
+        return QKeySequence(static_cast<QKeySequence::StandardKey>(var.toInt()));
+    return QKeySequence::fromString(var.toString());
+}
+
+QString StringUtilsInternal::shortcutToText(const QVariant &shortcut)
+{
+    const auto seq = variantToKeySequence(shortcut);
+
+    if (seq.isEmpty())
+        return {};
```

Comment:
Below we return a string and here an onbject(?). Is that intentional?

Reply:
This just returns a default constructed value (ie. `QString()` here)